TossClone/Assets.xcassets/TalkTableViewHeader.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  TalkTableViewHeader.swift
+//  TossClone
+//
+//  Created by 송시원 on 2022/07/11.
+//
+
+import UIKit
+
+class TalkTableViewHeader: UIView {
```

Comment:
final 을 적용하면 더이상 상속되지 않는다는 것을 명시할 수 있어요 ✨ 

```suggestion
final class TalkTableViewHeader: UIView {
```

Reply:
오 헤더는 더이상 상속하지 않아도 되니까 붙여주면 좋겠군요!